### PR TITLE
Added Prometheus WG info to list of SIG meetings

### PR DESCRIPTION
**Pull request by trask**
Same as #970 which was stuck since it was on a local repository branch.

EDIT: the doc hasn't been transferred to [cncf-opentelemetry-governance@lists.cncf.io](mailto:cncf-opentelemetry-governance@lists.cncf.io) yet, I've added that as a TODO in https://github.com/open-telemetry/community/issues/1147

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,6 +147,7 @@ Communications (Website, etc.)|Every other week on Monday at 10:00 PT|[Google Do
 End-User WG|Every other week on Thursday at 10:30 PT|[Google Doc](https://docs.google.com/document/d/1FqEo2PfHPP8iZX_W-C3YgXFUjhfwmAhoREUE8ZJPJfc/)||
 eBPF|Every week on Thursday at 10:30 PT|[Google Doc](https://docs.google.com/document/d/13GK915hdDQ9sUYzUIWi4pOfJK68EE935ugutUgL3yOw)|[Slack](https://cloud-native.slack.com/archives/C02AB15583A)|
 Agent Management WG|Every other week on Tuesday at 11:00 PT|[Google Doc](https://docs.google.com/document/d/19WA5-ex8rNFIBIyVb5VqMXfWNmUQwppGhN8zBeNG0f4)|[Slack](https://cloud-native.slack.com/archives/C02J58HR58R)|
+Prometheus Interoperability WG|Every week on Wednesday at 08:00 PT|[Google Doc](https://docs.google.com/document/d/19bnXziPn2MZ9wO6684UoI4D-LCjGL5bTJkGhux29bx8/edit#)|[Slack](https://cloud-native.slack.com/archives/C01LSCJBXDZ)|
```
Comment (trask) [inline README.md:150]: @open-telemetry/wg-prometheus it looks like meetings are now every other week? just confirming, and I will update.

also, the doc hasn't been transferred to cncf-opentelemetry-governance@lists.cncf.io yet, I've added that as a TODO in #1147

Comment (dashpole) [reply README.md:150]: Yes, meetings are currently every other week

Comment (trask) [reply README.md:150]:
```suggestion
Prometheus Interoperability WG|Every other week on Wednesday at 08:00 PT|[Google Doc](https://docs.google.com/document/d/19bnXziPn2MZ9wO6684UoI4D-LCjGL5bTJkGhux29bx8/edit#)|[Slack](https://cloud-native.slack.com/archives/C01LSCJBXDZ)|
```

File: README.md [hunk 2]
```diff
@@ -147,6 +147,7 @@ Communications (Website, etc.)|Every other week on Monday at 10:00 PT|[Google Do
 End-User WG|Every other week on Thursday at 10:30 PT|[Google Doc](https://docs.google.com/document/d/1FqEo2PfHPP8iZX_W-C3YgXFUjhfwmAhoREUE8ZJPJfc/)||
 eBPF|Every week on Thursday at 10:30 PT|[Google Doc](https://docs.google.com/document/d/13GK915hdDQ9sUYzUIWi4pOfJK68EE935ugutUgL3yOw)|[Slack](https://cloud-native.slack.com/archives/C02AB15583A)|
 Agent Management WG|Every other week on Tuesday at 11:00 PT|[Google Doc](https://docs.google.com/document/d/19WA5-ex8rNFIBIyVb5VqMXfWNmUQwppGhN8zBeNG0f4)|[Slack](https://cloud-native.slack.com/archives/C02J58HR58R)|
+Prometheus Interoperability WG|Every other week on Wednesday at 08:00 PT|[Google Doc](https://docs.google.com/document/d/19bnXziPn2MZ9wO6684UoI4D-LCjGL5bTJkGhux29bx8/edit#)|[Slack](https://cloud-native.slack.com/archives/C01LSCJBXDZ)|
```
Comment (arminru) [inline README.md:150]: @alolita can you please transfer ownership of the meeting notes doc to [cncf-opentelemetry-governance@lists.cncf.io](mailto:cncf-opentelemetry-governance@lists.cncf.io)?


see instructions in https://github.com/open-telemetry/community/issues/1147:

> ## Process to transfer ownership
>
> 1. Click the share button 
> 2. Invite cncf-opentelemetry-governance at lists.cncf.io to be an editor
> 3. Click the share button again
> 4. Click on the arrow next to the governance committee email and select "Transfer Ownership" 
> 5. Click "Send Invitation"

**Review by arminru — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (arminru) [reply README.md:150]: @alolita please take a look 🙂

**Review by dyladan — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (trask) [reply README.md:150]: going to merge this, will follow-up with @alolita

**Pull request merged**